example_test.go

```diff
@@ -231,6 +231,26 @@ func ExampleSelectRequest() {
 	// response is [{{} 1111 hello world}]
 }
 
+func ExampleSelectRequest_spaceAndIndexNames() {
+	isLess, err := test_helpers.IsTarantoolVersionLess(3, 0, 0)
```

Comment:
Really, there is no difference (because the string will just be resolved in case of Tarantool version < 3.0.0). Do I need to delete all these examples (because there is no real educational value)?

Reply:
You could left examples if we have not any examples for such requests types.

Reply:
Ok, updated.